### ISSUE TITLE
Icons aren't appearing

### ISSUE THREAD

**Issue by mbrockman1** [varjolintu/keepassxc#2]
Icons in the preference pane and the mac global menu bar are not showing

* KeePassXC 2.2.0-browser-beta4
* macOS Sierra 10.12.5 installed through .DMG

**Comment by varjolintu** [comment]
I cannot reproduce this. I have exactly the same macOS version. Can you provide some screenshots?

**Comment by mbrockman1** [comment]
<img width="172" alt="screen shot 2017-07-06 at 10 22 16 am" src="https://user-images.githubusercontent.com/20664971/27915748-1feb05d8-6235-11e7-9a11-fb0a6ab7de6e.png">
<img width="104" alt="screen shot 2017-07-06 at 10 21 48 am" src="https://user-images.githubusercontent.com/20664971/27915747-1fe8d1b4-6235-11e7-8a64-4d21016afc2c.png">

**Comment by varjolintu** [comment]
That's strange. Does the latest version (beta-5) work? Or the official KeePassXC release?

**Comment by mbrockman1** [comment]
No, the official release works however.

**Comment by whisdol** [comment]
I see exactly the same issue, in both the Mac build (can't verify which version right now) and the Windows build (Beta 8, Windows 10).

**Comment by varjolintu** [comment]
@whisdol It's strange that the Windows also has this issue. It seems this is the only big bug left but because I cannot reproduce it (only under Linux and with Docker builds) I will need some help from other guys.

Can you take logs of any kind that could help solving this? Maybe running [Process Monitor](https://docs.microsoft.com/en-us/sysinternals/downloads/procmon) could help?

**Comment by whisdol** [comment]
![image](https://user-images.githubusercontent.com/5374166/30270313-73bc1a8a-96ec-11e7-8ca6-adedb33d3471.png)
Process Monitor shows that some apparently static paths are configured for the icons, maybe that helps.
I uploaded the Process Monitor output (as .csv and in the Process Monitor format) to WeTransfer: https://we.tl/wv6Um5mPre

**Comment by varjolintu** [comment]
Thank you very much! I guess the same static path problem exists with the Linux version which I am able to reproduce. I have already built images for RC1 but this bug would be so nice to solve before an actual release.

**Comment by varjolintu** [comment]
At this point I have figured out that when the browser extension support is enabled in KeePassXC it gives the following errors:
```
QCoreApplication::applicationDirPath: Please instantiate the QApplication object first
FilePath::DataPath: can't find data dir
```

When inspecting the issue I noticed that when this happens `QCoreApplication::applicationDirPath()` returns an empty string. So some object is being used before QApplication and I need to find what. This might be also related to Boost or libsodium which are both used as static libraries.
